faker_edu/__init__.py

```diff
@@ -136,6 +136,31 @@
     'Teaching Assistant',
 ]
 
+COLLEGETYPES = [
```

Comment:
We keep all lists sorted alphabetically to make it easier to maintain them.  Please sort this list.